getting-started/openshift/windows/installation.md

```diff
@@ -1,319 +0,0 @@
----
-title: Install an OpenShift 4 cluster on Windows nodes
```

Comment:
Did you really mean to delete this doc entirely?

Reply:
Yes, I left this copy when I "moved" it to https://docs.projectcalico.org/master/getting-started/windows-calico/openshift/installation